src/js/components/RangeInput/index.d.ts

```diff
@@ -5,7 +5,7 @@ export interface RangeInputProps {
   min?: number | string;
   max?: number | string;
   name?: string;
-  onChange?: ((...args: any[]) => void);
+  onChange?: ((event: React.ChangeEvent) => void);
```

Comment:
I think @taysea reverted this event on purpose, @taysea can you review and remind us what was the issue that it had to be reverted?

Reply:
There had been various typescript issues with certain onChange handlers (specifically Select I think), so to be safe with other components that hadn't been validated yet, @britt6612 and I reverted the types to "any". I don't believe RangeInput had caused any errors, but we had reverted it just to be safe since the "onChange" handler had been causing problems for various components.

I think if there is clear documentation that this is the correct type, then it can be added back in.